lib/Quantization/Quantization.cpp

```diff
@@ -112,7 +112,13 @@ class FunctionQuantizer : public FunctionConverter {
   /// weren't specifically marked as to-ignore with doNotQuantizeKinds_.
   bool canConvert(const Node &node) const override {
     auto kind = node.getKind();
-    if (!EE_.isOpSupported(kind, quantizationPrecision_)) {
+
+    // Do not quantize node if target quantization precision is not supported
+    // by the backend or it's a SaveNode.
+    // SaveNode is typically a copy instruction, so it should be supported by
+    // the backend, but we do not want to quantize it.
+    if (!EE_.isOpSupported(kind, quantizationPrecision_) ||
+        kind == Kinded::Kind::SaveNodeKind) {
```

Comment:
`TypeAToTypeBFunctionConverter::canConvert()` calls into `FunctionConverter::canConvert()` which returns false for `SaveNode` and `Placeholder`. I wonder why we weren't doing that here already. Is that a preferable solution to this, at least for consistency's sake? Otherwise perhaps we should change `canConvert()` to pure virtual.